packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -105,6 +105,10 @@ export interface ISemanticColors {
    * Background for warning messages.
    */
   warningBackground: string;
+  /**
+   * Foreground color for warning highlights
+   */
+  warningForeground: string;
```

Comment:
can you call this warningHighlight instead, since it's not for text or icons (which are what foregrounds are)

Reply:
Well it's not a text highlight either, it's a highlight on the item in the form of a yellow side bar. I wasn't entirely sure what to call it.

Reply:
warningHighlight is fine, I just don't want confusion because a "foreground" color tends to be paired with a "background" color, i.e. fg goes on a bg, but that isn't the case here (note that errorText is meant to stand alone above, not go on top of errorBg, whereas I will eventually add an errorFg slot that DOES go on top of errorBg)

Reply:
Noted, I'll make that change.